app/decorators/litigation_decorator.rb

```diff
@@ -21,7 +21,10 @@ def document_links
     return [] if model.documents.empty?
 
     model.documents.map do |document|
-      h.link_to document.name, document.url, target: '_blank'
+      h.link_to document.name,
+                document.url,
+                target: '_blank',
+                title: document.external? ? document.url : nil
```

Comment:
not sure why the title should be `document.url` here and not always `document.name`

Reply:
I guess it could be handy for users to see where those external links are pointing.

Reply:
ahh, ok